converter/COLLADA2GLTF/main.cpp

```diff
@@ -82,6 +82,7 @@ static const OptionDescriptor options[] = {
 	{ "h",              no_argument,        "-h -> help" },
 	{ "r",              no_argument,        "-r -> verbose logging" },
 	{ "e",				no_argument,		"-e -> embed resources (bin, shaders, available textures) in glTF file" },
+    { "P",              no_argument,        "-P -> pretty print glTF JSON" },
```

Comment:
Yes, default to pretty print and change the ugly-print name to whatever is standard.